webapp/packages/core-blocks/src/FormControls/baseFormControlStylesNew.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { css } from 'reshadow';
+
+export const baseFormControlStylesNew = css`
+  field {
+    box-sizing: border-box;
+    padding: 12px;
+    min-height: 54px;
+    max-width: 100%;
+    &[|small] {
+      grid-column: 1/-4;
+    }
+    &[|medium] {
+      grid-column: 1/-3
+    }
+    &[|large] {
+      grid-column: 1/-2;
+    }
+    &[|full] {
+      grid-column: 1/-1;
+    }
```

Comment:
seems all these styles make sense only if some component used inside `<Grid>`
so they can be described in relation to `<Grid>` styles
```
Grid {
    &> {
      grid-column: 1/-1;
    }
    &>[|small] {
      grid-column: 1/-4;
    }
    &>[|medium] {
      grid-column: 1/-3
    }
    &>[|large] {
      grid-column: 1/-2;
    }
}
```